doc/notebooks/grass_jupyter.ipynb

```diff
@@ -0,0 +1,238 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Improved Integration of GRASS and Jupyter\n",
+    "\n",
+    "As part of Google Summer of Code 2021, we've been working to shorten and simplify the launch of GRASS in Jupyter and imporve the map displays. You can find out more about the project and follow the progress on the [GRASS wiki page](https://trac.osgeo.org/grass/wiki/GSoC/2021/JupyterAndGRASS).\n",
+    "\n",
+    "In addition to simplifying the launch of *GRASS GIS* with `init()`, `grass.jupyter` has two main dislay classes, `GrassRenderer` and `InteractiveMap`. `GrassRenderer` displays PNG images created with the *GRASS GIS* renderer. `InteractiveMap` displays *GRASS GIS* rasters and vectors with [*folium*](http://python-visualization.github.io/folium/), a [*leaflet*](https://leafletjs.com/) library for *Python*.\n",
```

Comment:
The sentence about GrassRenderer is misleading. It really renders, not just displays. It is called rendered after all. As similar sentence in one of the following cells is much better. However, starting from the current sentence something about "using GRASS rendering engine in the background" might be good.

Reply:
I went with : "Using the GRASS rendering engine in the background, GrassRenderer creates maps as PNG images."